test/e2e/run_selinux_test.go

```diff
@@ -177,4 +177,13 @@ var _ = Describe("Podman run", func() {
 		Expect(session.OutputToString()).To(Equal(session1.OutputToString()))
 	})
 
+	It("podman run --privileged and --security-opt SELinux options", func() {
+		session := podmanTest.Podman([]string{"run", "-it", "--privileged", "--security-opt", "label=type:spc_t", "--security-opt", "label=level:s0", ALPINE, "cat", "/proc/self/attr/current"})
```

Comment:
Change the level to something like s0:c1,c2
And then grep for it, would improve your tests.

Reply:
Fixed